### DISCUSSION TITLE
Unable to apply for educational discount

### DISCUSSION THREAD

**Discussion by noah-s-k** [community/community#52288] [GitHub Education] [answered]
Greeting guys,
I've been trying to apply for the edu. programm recently, but I'm having a hard time with the aplication process. 

The following was the Rejection:
![image](https://user-images.githubusercontent.com/62564300/230684218-b478785e-d6f2-4393-9f05-47141890f67d.png)

1. Point: I'm unable to add / verify my student E-Mail due to it not beeing allowed to receive E-Mail from outside the school network. (security reasons) So I must use my private @gmx.de E-Mail. 

2. Point: The second and fourth line of my Studen-ID reference the name of the school. Once in  text form and once in using its logo. I even wrote a hand written note in english beside it, that explains which line contains what content. Still rejected.

3. Point: I know, I'm at home fighting with this system.

**Comment by samus-aran** [comment]
Hi @mrflolo 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

Let's see how we can assist:

1. Our currently most preferred piece of evidence is Academic email. We understand not all students or institutions have this but if you have selected a school that does use academic emails that we trust we often enforce this. If you find yourself in this position we would recommend you reach out to your teachers and acquire one. But this isn't possible then perhaps ask if you are applying to the correct school.

2. This shouldn't be an issue as the system is able to detect text and logos. However again it could be down to the differences of school spellings to your evidence and the school you've selected in the application process.

3. We do accept remote learners but lets try and resolve to first two above and see if being far from campus is an actual issue.

Let us know how you get on 👍

**Reply by coolacpc** [reply]
Tried all of the above as well, Nothing so far, not even support responding 😢

**Comment by noah-s-k** [comment]
Hello @samus-aran, 

Thanks for your reply. 
> However again it could be down to the differences of school spellings to your evidence and the school you've selected in the application process

I looked further into the application form after you wrote that, and this time found my school in the drop-drown menu. 
Problem was that the schooltype "Gemeinschaftsschule" was translated to "Community school" in the school name. This isn't wrong ... but I did find some schools when searching for the german term, so that inconsistancy was causing my problem in finding it.

So now I've applied to the right school. But after not even a minute my application was rejected. But other than before no actual reason was given. 
![image](https://user-images.githubusercontent.com/62564300/231259335-74eb618c-a0d7-4096-85af-5148b54037a8.png)
I read through the given article but found no answer to my specific case. I hope I can find help here.

Have a nice day

**Comment by noah-s-k** [comment]
Still struggling

**Reply by samus-aran** [reply]
Hi @mrflolo 👋 

Sorry I missed my notifications!

Excellent detective work on finding your school. Usually when students receive no information from the rejection reasons calls for additional pieces of evidence. In this case you are using a Student ID, I would suggest you acquire another piece of information on top of the Student ID. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps! Let us know how you get along 👍

**Reply by noah-s-k** [reply]
Hey, @samus-aran.

thanks for your reply. I did try applying with a different form of prove, but it threw the following error: 
![6F63B4E9-9BC0-4EBA-8953-6F8943E41104](https://user-images.githubusercontent.com/62564300/234237278-5872fc49-2874-425a-97e4-640927104e66.jpeg)

I tried applying with a “school attendance certificate”, a common form of prove in Germany. It was provided by my schools office, and has a Signature and Stamp at the bottom. 
We don’t really have any of the other things listed in the drop-down at my school here in Germany. 

If needed I can provide a version with my personal data blacked out.

Have a nice day

**Reply by noah-s-k** [reply]
I'm kinda stuck now. Don't really know what to try next ...

**Reply by noah-s-k** [reply]
I guess no answer is also an answer.  

I'm going to try wrangling with the school IT to get an unlocked student E-Mail, maybe they will break protocol once. And as far as I understood I don't need it longer than a few days. 
I don't see any other way of ever getting this to work without one when there's no "real person" looking at the documents. Countries and ways and looks of proves are too diffrent. 
In the end the understandable wanting for secure prove whilst keeping the personnel size down kills this project.

Have a nice day

**Reply by coolacpc** [reply]
same here.....

At this point, i'll just go apply for jetBrains Codespace, seems easier!

**Reply by noah-s-k** [reply]
Greetings once again, 

I managed to an unlocked E-Mail and now have the next locked door in front of me. 
The schools homepage lies under the [SCHOOLNAME].de domain. This domain is also listed in the drop-down menu as the verifiable domain. BUT all E-Mails in the school network run under [NAME]@[SCHOOLNAME].net. This is due to the system "iServ" we use. 

I hope the trusted domain can be changed in the near future, and maybe I'll be able to get an apply through. 

Have a nice day,
Noah

**Comment by Nuo1Ye** [comment]
I have an educational email that has also not been approved. I have submitted it more than ten times, but every time it gives me the feeling of being a robot, I reply foolishly. I have provided various proofs as required, and I feel very disappointed

**Reply by coolacpc** [reply]
@Nuo1Ye It is approved by a robot... i THINK it gets reviewed by a person when you successfully apply and awating for next step

**Reply by Nuo1Ye** [reply]
Who knows, I have already applied for the JetBrains Codespace, and I think the review of Github is very foolish, and I can never tell people exactly what they need. Application after application, failure after failure, this feeling is very bad, so I am ready to give up the application because it does not help students correctly

**Reply by coolacpc** [reply]
i did the same thing, i mentioned above that  my chances are better with JetBrains Codespace and yes. it feels bad, same thing happened to me 😞

**Comment by samus-aran** [comment, marked as answer]
Closing this discussion in favour to this [one ](https://github.com/orgs/community/discussions/55619) requesting for the change of school domain.
